web/server/entrypoint.go

```diff
@@ -303,8 +303,7 @@ func (s *robotServer) serveWeb(ctx context.Context, cfg *config.Config) (err err
 	ctx = rpc.ContextWithDialer(ctx, rpcDialer)
 
 	processConfig := func(in *config.Config) (*config.Config, error) {
-		tlsCfg := config.NewTLSConfig(cfg)
```

Comment:
NewTLSConfig seems to only exist because of the if statement in https://github.com/viamrobotics/rdk/pull/4450/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17L964 and the fact that it's pulling in the original config instead of the config passed in. However, any subsequent/updated config will have the same fields as well. There does not seem to be anything else that is different than just creating a tls.Config inside ProcessConfig.